client/web/compose/src/components/ModuleFields/Editor/User.vue

```diff
@@ -319,7 +328,7 @@ export default {
     getUserByIndex (index = 0) {
       const userID = this.field.isMulti ? this.value[index] : this.value
       if (userID) {
-        return this.findByID(userID) || {}
+        return this.findByID(userID).userID || {}
```

Comment:
if it doesn't find the user it will be undefined, so same as above applies.
Also why the `|| {}`, if what you're returning is a userID?